apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -61,14 +71,31 @@ export const CompanyReview = ({
           reviewerId={reviewerId}
           reviewerProfilePicture={reviewerProfilePicture}
         />
-
         <Text color="gray-500" variant="sm">
           &bull;
         </Text>
-
         <Text color="gray-500" variant="sm">
           {reviewedAt}
         </Text>
+
+        {studentId === reviewerId && (
+          <div className="ml-auto">
+            <Tooltip>
+              <TooltipTrigger aria-label="Edit Review">
+                <Link
+                  to={generatePath(Route['/profile/work/:id/review/edit'], {
+                    id: workExperiencesId!,
```

Comment:
With the other changes, `workExperiencesId` should always be a string and so we don't need to use ! to make it non-null